cg/constants/pacbio.py

```diff
@@ -2,13 +2,15 @@
 
 
 class PacBioDirsAndFiles:
-    ADAPTER_REPORT: str = "adapter.report.json"
     BASECALLING_REPORT: str = "ccs.report.json"
```

Comment:
we don't need this anymore, right?

Reply:
no, we will get all those metrics from the other CCS report. I will remove it in the parsing refactoring (so next PR)